stl/inc/xstring

```diff
@@ -2282,6 +2282,17 @@ public:
         return _BUF_SIZE <= _Myres;
     }
 
+    constexpr void _Activate_SSO_buffer() {
+        // begin the lifetime of the array elements (e.g., before copying into them)
+#if _HAS_CXX20
+        if (_STD is_constant_evaluated()) {
+            for (size_type _Idx = 0; _Idx < _BUF_SIZE; ++_Idx) {
+                _Bx._Buf[_Idx] = value_type();

```

Comment:
No change requested: At first I thought we should say `_Elem()` like everywhere else, then I saw that we don't have that type here.